build_tools/cmake/test.sh

```diff
@@ -37,19 +38,16 @@ declare -a label_exclude_args=(
   ^driver=vulkan$
   ^nokokoro$
 
-  # TODO(b/151445957) Enable the python tests when the Kokoro VMs support them.
-  # See also, https://github.com/google/iree/issues/1346.
   # Exclude all tests in a directory.
```

Comment:
Should we keep these comments or delete them now that we don't have an actual use case?

Reply:
I think leave them as examples of how to do it. It took me a while to figure out :-D